gnosis/safe-react
# fix: Disable queued Tx interaction while replacement Tx is pending

## What it solves
Resolves #3359 

## How this PR fixes it

We now take the pending state of a transaction into account to decide whether their replacement transaction should be disabled or not

## How to test it

1. Open the Safe
2. Queue a Transaction
3. Reject the queued Transaction
4. Execute one of the transactions and confirm in Wallet
5. Observe that the other transaction stays disabled while the executed one is pending
6. Execute one of the transactions and reject in Wallet
7. Observe that the other transaction becomes available again

## Screenshots
![Screenshot 2022-02-02 at 16 36 35](https://user-images.githubusercontent.com/5880855/152185886-022d7456-f4d6-4301-9ce6-85a61fd20e73.png)
